Software/Middlewares/Third_Party/LoRaWAN/Core/lora.c

```diff
@@ -483,6 +483,17 @@ void LORA_Init (LoRaMainCallback_t *callbacks, LoRaParam_t* LoRaParam )
       /* Initialise LoRa to Geofence region */
 			LoRaMacInitialization( &LoRaMacPrimitives, &LoRaMacCallbacks, current_loramac_region );
 
+// source: https://github.com/Lora-net/LoRaMac-node/issues/742#issuecomment-492231231
+#if defined( REGION_US915 )
```

Comment:
Make the masking happen only at run time when using US915 and AU915 frequencies, instead of at compile time

Reply:
fixed in e73dc1f